package.json

```diff
@@ -63,7 +63,7 @@
     "expect.js": "0.3.1",
     "jsdoc": "3.6.3",
     "jsdoc-template": "https://github.com/braintree/jsdoc-template",
-    "mocha": "6.2.2",
+    "mocha": "7.0.1",
```

Comment:
Why the Mocha update in this PR?

Reply:
Ou, it seems `renovate-bot` update some dependencies. Maybe I used a bad branch name and that is the reason. What are the conventions?